react-app-rewired/index.ts

```diff
@@ -335,7 +335,8 @@ const reactAppRewireConfig = {
       _.merge(config, {
         resolve: {
           alias: {
-            'ethers/lib/utils': 'ethers/lib/utils.js',
+            'ethers/lib/utils': 'ethers5/lib/utils.js',
+            'ethers/lib/utils.js': 'ethers5/lib/utils.js',
```

Comment:
For packages still using ethers v5 i.e @cowprotocol/app-data and hdwallet packages (to be migrated to v6 too)